src/sentry/integrations/github/webhook.py

```diff
@@ -71,10 +72,30 @@ def __call__(self, event):
 
 class InstallationEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#installationevent
-    def _handle(self, event, organization, repo):
+    def __call__(self, event):
+        installation = event['installation']
+        if installation and event['action'] == 'deleted':
+            integration = Integration.objects.get(
+                external_id=installation['id'],
+                provider='github',
+            )
+
+            self._handle(event, integration)
+
+    def _handle(self, event, integration):
```

Comment:
i'd maybe rename this to something like `_handle_delete`? or you could also rename `InstallationEventWebhook` to `InstallationDeleteWebhook`? just to make it clear that deletion is the only event it's currently handling